execution-frameworks/Invoke-AtomicRedTeam/Invoke-AtomicRedTeam.ps1

```diff
@@ -0,0 +1,162 @@
+<#
+.SYNOPSIS
+This script will iterate over the Atomic Red Team yaml files, create objects for each test.
+The aim is to allow defenders to excercise MITRE ATT&CK Techniques to test defenses.
+
+
+Function: Invoke-AtomicRedTeam
+Author: Casey Smith @subTee
+License:  http://opensource.org/licenses/MIT
+Required Dependencies: powershell-yaml , Install-Module powershell-yaml #https://github.com/cloudbase/powershell-yaml
+Optional Dependencies: None
+Version: 1.0
+
+.DESCRIPTION
+Create Atomic Tests from yaml files described in Atomic Red Team. https://github.com/redcanaryco/atomic-red-team
+
+.EXAMPLE
+Convert Single Yaml File to Technique Object
+$T1117 = Get-AtomicTechnique -Path ..\..\atomics\T1117\T1117.yaml
+.EXAMPLE
+Get the Atomic Tests For A Given Technique
+Get-AtomicTest $T1117
+
+.NOTES
+This script converts Atomic Tests Expressed in YAML into PowerShell Objects.
+
+.LINK
+Blog: http://subt0x11.blogspot.com/2018/08/invoke-atomictest-automating-mitre-att.html
+Github repo: https://github.com/redcanaryco/atomic-red-team
+
+#>
+
+function Get-AtomicTechnique {
+[CmdletBinding()]
+Param(
+
+	[string]
+	$Path
+)
+# Returns A HashTable For Each File Passed In
+BEGIN { }
+PROCESS {
+		foreach ($File in $Path)
+		{
+			$parsedYaml = (ConvertFrom-Yaml (Get-Content $File -Raw ))
+			Write-Output $parsedYaml
+		}
+}
+END { }
+
+}
+
+function Get-AtomicTest{
+[CmdletBinding()]
+Param(
+	[System.Collections.Hashtable]
+	$AtomicTechnique
+)
+BEGIN{}
+PROCESS {
+	foreach ($Technique in $AtomicTechnique)
+	{
+
+		$AtomicTest = $Technique.atomic_tests
+
+		foreach ($Test in $AtomicTest)
+		{
+			#Only Process Windows Tests For Now
+			if ( !($Test.supported_platforms.Contains('windows')) ){ return }
```

Comment:
Your spacing in between the if parentheses is inconsistent. Sometimes you have a space trailing the `If` and sometimes you have none. Sometimes you have `space between parentheses and sometimes you have none.

For readability - I would recommend unifying the style.